SyllabusPlusSchedulerService/PublicApiWrapper/SessionManagementWrapper.cs

```diff
@@ -62,6 +62,34 @@ public void DeleteSessions(Guid sessionId)
         }
 
         /// <summary>
+        /// Gets a folder by ID
+        /// </summary>
+        /// <param name="folderId">Folder ID</param>
+        public Folder GetFolderById(Guid folderId)
+        {
+            return this.sessionManagement.GetFoldersById(this.authentication, new Guid[] { folderId }).First<Folder>();
```

Comment:
If it does throw, it will get caught by an outer try-catch. Bryan and I agreed this can be improved in a future update.